sormas-backend/src/main/java/de/symeda/sormas/backend/vaccination/VaccinationService.java

```diff
@@ -244,6 +246,24 @@ private boolean isVaccinationRelevant(Vaccination vaccination, Date... relevance
 		return false;
 	}
 
+	public List<Vaccination> getVaccinationListForACase(Case caze) {
```

Comment:
```suggestion
	public List<Vaccination> getRelevantVaccinationsForCase(Case caze) {
```

This method should already filter the vaccinations by relevance, this should not be outsourced to the method in VaccinationFacadeEjb; I can't think about any scenario in which we want to retrieve vaccinations without relevance here. i.e., the `isVaccinationRelevant` method needs to be called here already.